### PR TITLE
8.0 requester validation

### PR DESCRIPTION
**Pull request by yvaucher**

### CODE REVIEW

**Comment by lepistone** [comment]
:+1:

**Comment by gurneyalex** [comment]
:+1:

File: logistic_budget/view/sale_order.xml [hunk 1]
```diff
@@ -40,6 +40,17 @@
 
         <xpath expr="//notebook" position="after">
           <div class="oe_clear"/>
+            <div colspan="4">
```
Comment (yvaucher) [inline logistic_budget/view/sale_order.xml:44]: This seems misplaced. Should be in Validation group

**Comment by yvaucher** [comment]
Fixed and rebased

**Pull request merged**